pkg/porter/list.go

```diff
@@ -214,31 +214,32 @@ func NewDisplayRun(run storage.Run) DisplayRun {
 }
 
 // ListInstallations lists installed bundles.
-func (p *Porter) ListInstallations(ctx context.Context, opts ListOptions) ([]storage.Installation, error) {
+func (p *Porter) ListInstallations(ctx context.Context, opts ListOptions) (DisplayInstallations, error) {
 	ctx, log := tracing.StartSpan(ctx)
 	defer log.EndSpan()
 
 	installations, err := p.Installations.ListInstallations(ctx, opts.GetNamespace(), opts.Name, opts.ParseLabels())
-	return installations, errors.Wrap(err, "could not list installations")
-}
-
-// PrintInstallations prints installed bundles.
-func (p *Porter) PrintInstallations(ctx context.Context, opts ListOptions) error {
-	installations, err := p.ListInstallations(ctx, opts)
 	if err != nil {
-		return err
+		return nil, errors.Wrap(err, "could not list installations")
 	}
 
 	var displayInstallations DisplayInstallations
 	for _, installation := range installations {
-		di, err := p.generateDisplayInstallation(ctx, installation)
-		if err != nil {
-			return err
-		}
+		di := NewDisplayInstallation(installation)
```

Comment:
I moved the conversion of installations to display installations so that we could test these functions. In other parts of Porter we do the same thing but I guess this function is old enough it was doing the conversion in the print function still.